lib/ios/RCTHelpers.m

```diff
@@ -127,7 +127,13 @@ + (NSMutableDictionary *)textAttributesFromDictionary:(NSDictionary *)dictionary
 
 + (NSMutableDictionary *)textAttributesFromDictionary:(NSDictionary *)dictionary withPrefix:(NSString *)prefix
 {
-	return [self textAttributesFromDictionary:dictionary withPrefix:prefix baseFont:[UIFont systemFontOfSize:[UIFont systemFontSize]]];
+#if !(TARGET_OS_TV)
+	CGFloat systemFontSize = [UIFont systemFontSize];
+#else
+	CGFloat systemFontSize = 20;
```

Comment:
where does `20` come from? Is there a better way of providing that value?

Reply:
Totally plucked out of thin air. tvOS doesn't provide a value for `systemFontSize` so we kinda just have to make one up.

Reply:
fair